src/graphnet/models/gnn/dynedge.py

```diff
@@ -170,7 +170,12 @@ def forward(self, data: Data) -> Tensor:
 
 
 class DynEdge_V2(GNN):
-    def __init__(self, nb_inputs, layer_size_scale=4):
```

Comment:
The `Model` class already has the option to do coarsening, and the implementation in `DynEdge` and this proposed change to `DynEdge_V2` does the coarsening before any actual convolutional operations, so I think we could leave this functionality in the `Model` class. (That's at least what I am proposing in #302.) Of course, if we had a `GNN` model that relied on coarsening "in the middel" of the convolutional operations, that would be a different story, but that's not the case here, as far as I can tell.